backend/app/main.py

```
@@ -1,16 +1,45 @@
 async def lifespan(app: FastAPI):
+    # アプリケーション起動時
     models.Base.metadata.create_all(bind=engine)
+    
+    # カスタムサイトテーブルのURLカラムを更新（必要な場合）
+    try:
+        from sqlalchemy import text
+        with engine.connect() as conn:
+            # PostgreSQLの場合
+            try:
+                conn.execute(text("""
+                    ALTER TABLE custom_sites 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)